Config.ts

```diff
@@ -24,7 +24,6 @@ class Config implements Readonly<MutableConfig> {
     private static validate(config: Config) {
         assert(config.unprocessed_label.length > 0, "unprocessed_label can't be empty");
         assert(config.processed_label.length > 0, "processed_label can't be empty");
-        assert(config.personal_label.length > 0, "personal_label can't be empty");
```

Comment:
In line 36 the config object has default value for this field, so it's in fact not required to be filled in the spreadsheet.
But you are right that this field is not used anywhere and I actually have no plan to use it.
Since you are here, could you help to remove this field everywhere?

Reply:
Sure, done.

Reply:
On a quick try-out, this does cause the spreadsheet to error until "personal label" is removed from the config page. Is that what you want, versus just ignoring the value?

Reply:
Good catch.  Let's change line 74 to 
```ts
console.error(`Invalid config: ${name}`);
```

Reply:
Done

Reply:
Thanks!